src/Input/ToggleSwitch/ToggleSwitch.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { ToggleSwitch, ToggleSwitchProps } from './ToggleSwitch';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+import EyeIcon from '../../General/Icon/components/EyeIcon';
+import EyeSlashedIcon from '../../General/Icon/components/EyeSlashedIcon';
+
+export default {
+  title: 'Input/ToggleSwitch',
+  component: ToggleSwitch,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+} as Meta;
+
+const Template: Story<ToggleSwitchProps> = args => <ToggleSwitch {...args} />;
+
+export const Interactive = Template.bind({});
+Interactive.args = {
+  defaultActive: false,
+};
+
+export const ToggleSwitchWithIcons = Template.bind({});
```

Comment:
I think better to write a full story here. Like export 

```
const ToggleSwitchWithIcons: Story<ToggleSwitchProps> = () => <ToggleSwitch
  active={...}
  inactive={...}
/>
```

Otherwise the code for the example looks useless.

Reply:
Oh didn't see the example code. Thanks for taking a look!!